pixels-core/src/main/java/io/pixelsdb/pixels/core/PixelsWriterImpl.java

```diff
@@ -519,6 +520,14 @@ private void writeRowGroup()
                     byte[] rowGroupBuffer = writer.getColumnChunkContent();
                     physicalWriter.append(rowGroupBuffer, 0, rowGroupBuffer.length);
                     writtenBytes += rowGroupBuffer.length;
+                    int alignment = Integer.parseInt(ConfigFactory.Instance().getProperty("row.group.align"));
+                    // add align bytes to make sure the column size is the multiple of `alignment`
+                    if(rowGroupBuffer.length % alignment != 0) {
+                        int alignByte = alignment - rowGroupBuffer.length % alignment;
+                        byte[] emptyArray = new byte[alignByte];
```

Comment:
Rename it to chunkPaddingBuffer and make it a private static final member of the class. Its length can be chunkAlignment.

chunkPaddingBuffer and chunkAlighment can be initialized in a static domain in the class.